From db2a137cbb556509c3d64456914a1353764a4dd7 Mon Sep 17 00:00:00 2001
From: Cody Boyko <cody.boyko2@ontario.ca>
Date: Mon, 24 Sep 2018 13:23:15 -0400
Subject: [PATCH] [#4336] Remove html5 shim from stats extension.

html5.js was removed in commit #3705 but still referenced in
the stats extension. Currently this causes an error when the page
loads in CKAN 2.8 installs.

Removing this will remove support for older IE versions which
will only cause some issues for CKAN 2.8 users with the old
bootstrap 2 theme being used.
---
 ckanext/stats/templates/ckanext/stats/index.html | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/ckanext/stats/templates/ckanext/stats/index.html b/ckanext/stats/templates/ckanext/stats/index.html
index ef333c2ba3..45f4ad2846 100644
--- a/ckanext/stats/templates/ckanext/stats/index.html
+++ b/ckanext/stats/templates/ckanext/stats/index.html
@@ -189,11 +189,5 @@ <h2 class="module-heading"><i class="fa fa-bar-chart-o fa-medium"></i> {{ _('Sta

 {% block scripts %}
   {{ super() }}
-{#
-Hellish hack to get excanvas to work in IE8. We disable html5shiv from
-overriding the createElement() method on this page.
-See: http://stackoverflow.com/questions/10208062/using-flot-with-bootstrap-ie8-incompatibility
-#}
-{% resource "vendor/block_html5_shim" %}
-{% resource "ckanext_stats/stats" %}
+  {% resource "ckanext_stats/stats" %}
 {% endblock %}
